addons/web/static/tests/views/list_tests.js

```diff
@@ -9059,6 +9059,39 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('control panel buttons in multi editable grouped list views', async function (assert) {
+        assert.expect(5);
+
+        var list = await createView({
+            View: ListView,
+            model: 'foo',
+            data: this.data,
+            groupBy: ['foo'],
+            arch: '<tree multi_edit="1">' +
+                        '<field name="foo"/>' +
+                        '<field name="int_field"/>' +
+                    '</tree>',
+        });
+
+        assert.containsNone(list, '.o_data_row', "all groups should be closed");
+        assert.isVisible(list.$buttons.find('.o_list_button_add'),
+            "should have a visible Create button");
+
+        await testUtils.dom.click(list.$('.o_group_header:first'));
+        assert.isVisible(list.$buttons.find('.o_list_button_add'),
```

Comment:
don't forget sanity checks:
`assert.containsN(list, '.o_data_row', ?, "first group should be opened);`